examples/rocksdb/README.md

```diff
@@ -27,3 +27,69 @@ cd examples/rocksdb/
 . ./venv/bin/activate
 ./campaign_rocksdb.py
 ```
+
+## Speedup stacks
+
+To run the speedup stack example, you need the following dependencies.
+
+- LLVM 3.7.1 or newer, compiled with BPF support (default=on)
+- Clang, built from the same tree as LLVM
+- cmake (>=3.1), gcc (>=4.7), flex, bison
+- LuaJIT, if you want Lua support
+- Optional tools used in some examples: arping, netperf, and iperf
+
+```
+# For Focal (20.04.1 LTS)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm12 llvm-12-dev libclang-12-dev python zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev arping netperf iperf
+
+# For Hirsute (21.04) or Impish (21.10)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm12 llvm-12-dev libclang-12-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev arping netperf iperf
+
+# For Jammy (22.04)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm14 llvm-14-dev libclang-14-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev libdebuginfod-dev arping netperf iperf
+
+# For Lunar Lobster (23.04)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm15 llvm-15-dev libclang-15-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev libdebuginfod-dev arping netperf iperf libpolly-15-dev
+
+# For Mantic Minotaur (23.10)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm16 llvm-16-dev libclang-16-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev libdebuginfod-dev arping netperf iperf libpolly-16-dev
+
+# For Noble Numbat (24.04)
+sudo apt install -y zip bison build-essential cmake flex git libedit-dev \
+  libllvm18 llvm-18-dev libclang-18-dev python3 zlib1g-dev libelf-dev libfl-dev python3-setuptools \
+  liblzma-dev libdebuginfod-dev arping netperf iperf libpolly-18-dev
+
+# For other versions
+sudo apt-get -y install zip bison build-essential cmake flex git libedit-dev \
+  libllvm3.7 llvm-3.7-dev libclang-3.7-dev python zlib1g-dev libelf-dev python3-setuptools \
+  liblzma-dev arping netperf iperf
```

Comment:
probably make sense to integrate these complex dependencies management directly in the code of the klockstat? With something that check the os version from `/etc/os-release`?